build.gradle

```diff
@@ -1,14 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.2.51'
+
     repositories {
         jcenter()
         mavenCentral()
         google()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.2'
-    }
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version" }
```

Comment:
I think it's not relate to current issue

Reply:
I check and update PR in several hours.